docs/primer.md

```diff
@@ -1,15 +1,322 @@
+
+- Individuals at high risk of infection given exposure (e.g. elderly, some immunocompromised) may be more likely to get vaccinated than the general public. In this case, the realized value of $\theta_{\lambda} = \frac{\lambda_{FV}}{\lambda_{FU}}$ is closer to 1 than it would be if it only captured immunological phenomena.
+
+- Individuals who get vaccinated may be more likely to use at-home treatments for mild illness which help prevent progression to ARI. In this case, the realized value of $\theta_{\pi} = \frac{\pi_{FV}}{\pi_{FU}}$ is closer to 0 than it would be if it only captured immunological phenomena.
+
+These examples show that a purely immunological interpretation of VE could under- or overestimate the impact of vaccination, depending on which conditions and behaviors correlate with vaccination.
+
+### Probability of Being Tested Differs by Disease and Vaccination Status
+
+To this point, we have tacitly assumed that everyone with ARI who seeks care is tested for the causal pathogen. But in practice, ARI symptoms may manifest differently based on the causal pathogen and/or vaccination status, leading to different probabilities of a clinician deciding to administer a test. If we reimpose all previous assumptions but introduce ζ as the probability of being tested, we have:
+
+$$
+\begin{align*}
+\mathrm{VE} &= 1 - \frac{\left( \varepsilon_{FV}tvP\left\lbrack \left( 1 - \varphi \right)\lambda_{FU}\pi_{FU} + \varphi\lambda_{FV}\pi_{FV} \right\rbrack\mu_{V}\zeta_{FV} \right)\left( \varepsilon_{NU}t\left( 1 - v \right)P\lambda_{NU}\pi_{NU}\mu_{U}\zeta_{NU} \right)}{\left( \varepsilon_{FU}t\left( 1 - v \right)P\lambda_{FU}\pi_{FU}\mu_{U}\zeta_{FU} \right)\left( \varepsilon_{NV}tvP\lambda_{NV}\pi_{NV}\mu_{V}\zeta_{NV} \right)} \\
+&= 1 - \frac{\zeta_{FV}\zeta_{NU}}{\zeta_{FU}\zeta_{NV}}\left( 1 - \varphi + \varphi\theta \right)
+\end{align*}
+$$
+
+Thus, unless the effect of vaccination on the probability of being tested is identical for both the focal and non-focal pathogens, $\frac{\zeta_{FV}}{\zeta_{FU}} = \frac{\zeta_{NV}}{\zeta_{NU}}$, then testing probability also contributes to bias in $1 - \mathrm{OR}$.
+
+## Conclusions
+
+$1 - \mathrm{OR}$ from a TND study is an unbiased estimator of VE only under a restrictive and unrealistic set of assumptions. Violating these assumptions introduces biases which can be understood in isolation, but which may interact with one another in complicated ways. Nonetheless, some themes emerge:
+
+- The transmission and/or disease caused by the focal vs. non-focal pathogens need not be identical, but they must respond to vaccination identically (except for $\lambda$ and $\pi$), to avoid bias.
+- Dynamics cause bias not because the rates of observing TND quadrants changes through time, but because the _relative_ rates of observing vaccinated vs. unvaccinated TND quadrants change through time.
```

Comment:
```suggestion
- Dynamics cause bias not because the rates of observing TND quadrants change through time, but because the _relative_ rates of observing vaccinated vs. unvaccinated TND quadrants change through time.
```